README.md

```diff
@@ -4,13 +4,13 @@
 ## Social medias
 [Join our discord](https://discord.gg/8bHCNmhS7x) or [Telegram](https://t.me/web3swift) if you need support or want to contribute to web3swift development!
 
-![matter-github-swift](https://github.com/skywinder/web3swift/blob/develop/web3swift-logo.png)
-[![Web3swift CI](https://github.com/skywinder/web3swift/actions/workflows/ci.yml/badge.svg)](https://github.com/skywinder/web3swift/actions/workflows/ci.yml)
+![matter-github-swift](https://github.com/web3swift-team/web3swift/blob/develop/web3swift-logo.png)
+[![Web3swift CI](https://github.com/web3swift-team/web3swift/actions/workflows/macOS-12.yml/badge.svg)](https://github.com/web3swift-team/web3swift/actions/workflows/macOS-12.yml)
 [![Swift](https://img.shields.io/badge/Swift-5.4-orange.svg?style=flat)](https://developer.apple.com/swift/)
-[![Platform](https://img.shields.io/cocoapods/p/web3swift.svg?style=flat)](http://cocoapods.org/pods/web3.swift.pod)
-[![CocoaPods Compatible](https://img.shields.io/cocoapods/v/web3.swift.pod.svg?style=flat)](http://cocoapods.org/pods/web3.swift.pod)
+[![Platform](https://img.shields.io/cocoapods/p/web3swift?style=flat)](http://cocoapods.org/pods/web3swift)
+[![CocoaPods Compatible](https://img.shields.io/cocoapods/v/web3swift?style=flat)](http://cocoapods.org/pods/web3swift)
```

Comment:
Platform and CocoaPods Compatible visuals breaks sometimes. And it looks like it's not related to web3swift project only:

<img width="308" alt="Screenshot 2022-12-13 at 19 34 54" src="https://user-images.githubusercontent.com/36865532/207404507-dbd33d07-a87e-42a8-83c9-2a869a590ad4.png">

_This 2.2.1 version is a screenshot of the version of web3.swift.pod library, not web3swift._
<img width="185" alt="Screenshot 2022-12-13 at 19 32 39" src="https://user-images.githubusercontent.com/36865532/207404519-4d720083-6e83-4859-afde-4865b30d6f5c.png">

Reply:
Lol